dora/core/common/pom.xml

```diff
@@ -141,6 +141,11 @@
       <artifactId>netty-tcnative-boringssl-static</artifactId>
       <version>2.0.56.Final</version>
     </dependency>
+    <dependency>
```

Comment:
nit: try to keep alphabetical order in dependency

also use Dependency Manager for versions

Reply:
done